addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/handler/AbstractNetatmoThingHandler.java

```diff
@@ -156,4 +157,18 @@ protected String getId() {
         }
     }
 
+    protected void updateProperties(Integer firmware, String modelId) {
+        Map<String, String> properties = editProperties();
+        if ((firmware != null || (modelId != null)) && (properties.get(Thing.PROPERTY_VENDOR) == null)) {
```

Comment:
I am wondering why you check `firmware` and `modelId` against `null` in this case?

Reply:
The idea is to set vendor only on things that are linked to a netatmo device, that means not on a home or a person for example.